xtable-core/src/test/java/org/apache/xtable/ITConversionController.java

```diff
@@ -483,55 +605,6 @@ public void testTimeTravelQueries(String sourceTableFormat) throws Exception {
     }
   }
 
-  private static List<String> getOtherFormats(String sourceTableFormat) {
-    return Arrays.stream(TableFormat.values())
-        .filter(format -> !format.equals(sourceTableFormat))
-        .collect(Collectors.toList());
-  }
-
-  private static Stream<Arguments> provideArgsForPartitionTesting() {
```

Comment:
A lot of the changes to this file are just moving methods around. Let's avoid doing this going forward unless it is required or makes the code a lot easier to read. It adds more overhead to the review for me to go line by line through this to check what changes there are to the test setups.